test/nodejs/test_file.rb

```diff
@@ -54,6 +54,26 @@ def test_read_each_line
       lines << line
     end
     assert_equal(lines.length, 2)
+    assert_equal(lines[0], 'one')
+    assert_equal(lines[1], 'two')
+  end
+
+  def test_readlines
+    File.write('tmp/quz', "one\ntwo")
+    file = File.new('tmp/quz', 'r')
+    lines = file.readlines do |line|
+    assert_equal(lines.length, 2)
+    assert_equal(lines[0], 'one')
```

Comment:
MRI keeps `"\n"` in the end of each line:
``` ruby
2.5.1 :001 > File.read('file')
 => "asd\nqwe\n"
2.5.1 :002 > File.open('file').readlines
 => ["asd\n", "qwe\n"]
```

Reply:
fixed